agent/conf/agent.properties

```diff
@@ -285,3 +285,6 @@ iscsi.session.cleanup.enabled=false
 
 # Enable manually setting CPU's topology on KVM's VM.
 # enable.manually.setting.cpu.topology.on.kvm.vm=true
+
+# host.cpu.manual.speed.mhz = int
+# Manually set the host CPU MHz, in cases where CPU scaling support detected value is wrong
```

Comment:
should we adhere to the convention of putting the explaining comment before the commented default value?

Reply:
```suggestion
# Manually set the host CPU MHz, in cases where CPU scaling support detected value is wrong
# host.cpu.manual.speed.mhz=0
```